src/popup/components/Plate.vue

```diff
@@ -1,7 +1,7 @@
 <template>
```

Comment:
In my opinion we need to remove the Plate component instead of tweaking it. There are only 2 places that uses it and the Figma designs does not use it anymore.